data/priyankarpal.json

```
@@ -2,28 +2,68 @@
   "name": "Priyankar Pal",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Avid Linux User **Web Dev**. Tech Enthusiast",
+  "bio": "Avid Linux User | **Web Dev** | Tech Enthusiast",
   "avatar": "https://github.com/priyankarpal.png",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "JAVA",
+    "SCSS"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/priyankarpal" },
+    { "icon": "FaGithub", "url": "https://github.com/priyankarpal" },
+    { "icon": "FaLinkedin", "url": "https://linkedin.com/in/priyankarpal" },
+    { "icon": "FaInstagram", "url": "https://instagram.com/priyankarpal1" }
+  ],
   "links": [
     {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/priyankarpal",
-      "icon": "FaTwitter"
+      "name": "Check My Latest Blogs",
+      "url": "https://priyankarpal.hashnode.dev",
+      "icon": "FaLink",
```

```suggestion
      "icon": "SiHashnode",
```
We have an icon for hashnode you could use.